pymobiledevice3/services/afc.py

```diff
@@ -772,7 +772,10 @@ def do_pull(self, args):
         def log(src, dst):
             self.poutput(f'{src} --> {dst}')
 
-        self.afc.pull(args.remote_path, args.local_path, callback=log, src_dir=self.curdir)
+        remote_path = str(pathlib.PurePosixPath(self.curdir).joinpath(args.remote_path))
+        if args.remote_path.endswith(posixpath.sep):
```

Comment:
Add comment that explains the rational behind it (that pathlib removes the `/`)

Reply:
done